views/search/logs.ejs

```diff
@@ -23,15 +23,14 @@
         <div class="col-md-12">
           <ul class="list-group search-results-adv">
             <li class="list-group-item" ng-if="searching">Searching...</li>
-            <a ng-href="/u/{{result.userName}}"
```

Comment:
I got rid of the hyperlink here, because linking to the user's reference page isn't really useful.